about/index.html

```diff
@@ -92,6 +92,42 @@ <h1> About OpenVDB</h1>
           </ul>
           </p>
 
+          <p>The library was first Open Sourced by the team at DreamWorks in 2013 with a strong
+          emphasis on efficient Level Set representations and tools. It has since seen many
+          additions which have broadened its scope and feature set. Some notable additions include:</p>
```

Comment:
This now sounds like there were no "notable additions" between the initial release 2012 and 2016 which is obviously not the case (covers up to version v4). Not sure how to address this though

Reply:
I can change it to say "recent notable additions"? Basically wanted to highlight the big features that have come outside of DreamWorks since the ASWF

Reply:
that works